src/exberry_int/integration_exberry.py

```diff
@@ -63,11 +63,13 @@ def integration_exberry_main(
     events: 'IntegrationEvents'):
 
     outbound_queue = asyncio.Queue()
+    pending_outbound_queue = asyncio.Queue()
```

Comment:
Rather than two queues for the two different priority levels, why not just use a priority queue?

https://docs.python.org/3/library/asyncio-queue.html#priority-queue

This gives you the ability to insert into queue with priority levels and trust that the dequeues (from the single solitary queue) are fetched in priority order.